docker/experimental/Dockerfile

```diff
@@ -46,7 +48,8 @@ WORKDIR /pytorch
 COPY torch_patches/ torch_patches/
 RUN bash -c "torch_pin=$(cat torch_patches/.torch_pin &); git fetch origin ${torch_pin:-master}; git checkout origin/${torch_pin:-master}"
 
-RUN USE_CUDA=0 python setup.py bdist_wheel
```

Comment:
do we want to set `USE_CUDA=0` as a defualt config? otherwise we might accidently build with `USE_CUDA=1` which is slower.

Reply:
What do you mean? `USE_CUDA=0` is there already. Do you mean this should be configurable? Do we ever want to build the PyTorch GPU wheel ourselves?

Reply:
oh I mean this `USE_CUDA=0` would only take effect in this command right? Say someone download this docker and want to do development within this docker, when they do `python setup.py instlall` for pytorch, it will compile with `USE_CUDA=1`. I am wondering if we should do
https://github.com/pytorch/xla/blob/fd4151bc9707b38b24bf10b81a95f5a4e33b196d/docker/Dockerfile#L21

Reply:
Yeah, that makes sense. Done.